sycl/source/detail/context_impl.hpp

```diff
@@ -177,6 +180,8 @@ class context_impl {
   std::map<std::pair<DeviceLibExt, RT::PiDevice>, RT::PiProgram>
       MCachedLibPrograms;
   mutable KernelProgramCache MKernelProgramCache;
+  // 0 - not supported, 1 - supported, 2 - check has not done yet
```

Comment:
It's better to have enum or a bool with human-readable values rather than a mere integer. Otherwise, these are magic numbers which aren't intuitive to understand.

Reply:
Replaced it by enum